### PR TITLE
Issue 316

### PR DESCRIPTION
**Pull request by disperse**
Changes to forms view to enable drag-and-drop reordering and enhancements to form skip handling and display.

### CODE REVIEW

File: src/views/Forms.vue [hunk 1]
```diff
@@ -37,38 +37,43 @@
...
+              </form-list-tile>
+            </draggable>
           </template>
         </v-data-table>
       </v-card>
     </v-flex>
     <v-dialog v-model="showImportForm" @formImported="onFormImported(importedForm)" max-width="50em">
       <form-import :form-type="importFormType"></form-import>
     </v-dialog>
+    <v-dialog v-model="showFormSkips">
```
Comment (wyattis) [inline src/views/Forms.vue:74]: Why not just use TrellisModal here?

Comment (wyattis) [reply src/views/Forms.vue:74]: Ah, I see. We need a way to access the title slot from the modal as well.

File: src/components/ClickToEdit.vue [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-      :append-icon="appendIcon"
-      :prepend-icon="internal.editing ? 'mdi-clear' : ''"
-      @click:prepend="resetEditorState"
-      @click:append="onClickAppend"
-      v-model="memText"
-      @keyup.enter="save"
-      class="min-text-field" />
-  </v-flex>
+  <v-text-field
+    :full-width="true"
```
[the remark this answers is not in the document]
Comment (disperse) [reply src/components/ClickToEdit.vue:3]: Changed this to:
```
  <v-text-field
    full-width
    flat
    hide-details
...
```

File: src/components/forms/FormListTile.vue [hunk 3]
```diff
@@ -71,9 +74,8 @@
       <v-checkbox v-model="memForm.isPublished" @change="save"></v-checkbox>
     </td>
     <td>
-      <v-btn icon @click="$emit('input', !value)">
-        <v-icon v-if="value">mdi-chevron-up</v-icon>
-        <v-icon v-else>mdi-chevron-down</v-icon>
+      <v-btn icon @click="$emit('toggleFormSkips', studyForm.form)">
+        <v-icon :class="{ 'primary--text': (studyForm.form.skips.length > 0) }">{{ icons.mdiArrowDecision }}</v-icon>
```
[the remark this answers is not in the document]
Comment (disperse) [reply src/components/forms/FormListTile.vue:78]: It seems to work as `<v-icon>mdi-arrow-decision</v-icon>` made the change.

File: src/components/skips/SkipEditor.vue [hunk 4]
```diff
@@ -37,7 +38,7 @@
     components: { SkipRow },
     props: {
       skips: Array as () => Skip[],
-      conditionTags: Array as () => ConditionTag[],
+      conditionTags: Array as () => ConditionTag[] | null,
```
[the remark this answers is not in the document]
Comment (disperse) [reply src/components/skips/SkipEditor.vue:41]: OK, probably needs to be part of a refactor in the future.

File: src/views/Forms.vue (hunk 1 above)
Comment (disperse) [reply src/views/Forms.vue:74]: I think we can extend Trellis Modal to have an additional title slot, didn't seem worth the time for now. We can revisit.

**Pull request merged**